src/plugins/intel_cpu/src/transformations/tpp/x64/op/eltwise.hpp

```diff
@@ -26,20 +27,25 @@ class EltwiseTPP : public modifier::TensorProcessingPrimitive {
 public:
     static bool is_supported(const std::shared_ptr<ov::Node>& node);
     bool visit_attributes(AttributeVisitor& visitor);
+    virtual OpDescTPP get_op_desc() const  = 0;
 };
 
 class BinaryEltwiseTPP : public EltwiseTPP {
 public:
     BinaryEltwiseTPP(libxsmm_meltw_binary_type op_type);
-    libxsmm_meltw_binary_type get_op_type() const { return m_op_type; }
-private:
+    OpDescTPP get_op_desc() const override { return OpDescTPP(m_op_type, m_flags); }
+    static libxsmm_bitfield get_broadcasting_flags(const ov::PartialShape& pshape_0, const ov::PartialShape& pshape_1);
+    static libxsmm_bitfield get_broadcasting_flags(const snippets::VectorDims& pshape_0, const snippets::VectorDims& pshape_1);
+
+protected:
+    libxsmm_bitfield m_flags;
     libxsmm_meltw_binary_type m_op_type;
```

Comment:
Can these attributes be changed after the node creation?
If they cannot, could we store just `OpDescTPP` in the attributes of the base class `EltwiseTPP` instead of flags, op_type in `BinaryEltwiseTPP` and op_type `UnaryEltwiseTPP`? If you need these values, you can just get them from `OpDescTPP`. Also the method `get_op_desc()` will return just `m_op_desc` (no need to create this object in each time of the method call)?

Reply:
I thought about this as well, but it won't work unfortunately. The thing is that the `m_flags` should always reflect broadcasting rules, and thus can be updated in any `validate_and_infer_types` call.
Of course we can update OpDesc every time, but this doesn't make much sense since `get_op_desc` is called only once in emitters. 
Moreover the present design enforces you to specify `libxsmm_meltw_unary_type` for every UnaryEltwiseTPP. If we store Opdescs in the base class, we could easily create UnaryEltwiseTPP using libxsmm_meltw_binary_type which should clearly be avoided.